### ISSUE TITLE
📝 Change mention of update.cloudflare.com to update.argotunnel.com

### ISSUE THREAD

**Issue by AllexVeldman** [cloudflare/cloudflared#690]
**Available Documentation**
https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/configuration/ports-and-ips/
mentions:
```
TCP port 443 (HTTPS)
IPs are those behind api.cloudflare.com and update.cloudflare.com *
```

**Suggested Documentation**
Change `update.cloudflare.com` to `update.argotunnel.com` or add it as a separate known host/ip.

**Additional context**
I'm using Istio to manage and monitor network traffic with ServiceEntry's added for the hosts/ips mentioned in the documentation.
Running cloudflared 2022.7.1, it tries to connect to `update.argotunnel.com` instead of `update.cloudflare.com`, which as expected is blocked.
```
ERR update check failed error="Get \"https://update.argotunnel.com?arch=amd64&clientVersion=2022.7.1&os=linux\": read tcp 10.244.1.28:58270->104.18.32.167:443: read: connection reset by peer"
```

**Comment by vijaytdh** [comment]
Since it is the same page - could I tack on updating the page to also mention the v2 endpoints:

```
region2.v2.argotunnel.com
region1.v2.argotunnel.com
```
We updated from `2022.5.0` to `2022.7.1` and starting seeing errors about connecting to IPs such as 198.41.192.37. I only found these domains because I grepped through the [code](https://github.com/cloudflare/cloudflared/blob/dd540af695448ea1bfef28762060fb98455d455d/component-tests/test_edge_discovery.py#L110) for the word region.

**Comment by DevinCarr** [comment]
I have added both of these changes to the docs change here: https://github.com/cloudflare/cloudflare-docs/pull/5061

Thanks for the notice! These were changed as part of the updated edge discovery to include IPv6 support.

**Comment by vijaytdh** [comment]
@DevinCarr  thanks for the quick response. Sorry I know this issue has been closed but would you be able to help clarify a few points:

You mentioned this was changed as part of IPv6 support. Does this mean that the non "v2" IP ranges associated with `region1.argotunnel.com` and `region2.argotunnel.com` are no longer required at all - since in the MR to update the docs they have been removed altogether? Or are they required if you are using IPv4?

**Comment by DevinCarr** [comment]
As part of the IPv6 changes included in cloudflared version 2022.7.0 (and onwards), we updated to use the new regions: `region1.v2.argotunnel.com` and `region2.v2.argotunnel.com`. The new regions are a superset of the original `region1.argotunnel.com` and `region2.argotunnel.com`. So you shouldn't need to remove the older IPv4 addresses, but you can update to include the new IPv4 addresses (and IPv6 addresses if available on your system) from the `*.v2.argotunnel.com` regions.

**Comment by vijaytdh** [comment]
@DevinCarr great thanks for the explanation, makes sense.
